administrator/components/com_installer/tmpl/downloadkeys/default.php

```diff
@@ -0,0 +1,122 @@
+							<tr>
+								<td style="width:1%" class="text-center">
+									<?php echo HTMLHelper::_('grid.checkall'); ?>
+								</td>
+								<th scope="col" class="nowrap">
+									<?php echo HTMLHelper::_('searchtools.sort', 'COM_INSTALLER_HEADING_UPDATESITE_NAME', 'update_site_name', $listDirn, $listOrder); ?>
+								</th>
+								<th scope="col" style="width:20%" class="d-none d-md-table-cell">
+									<?php echo HTMLHelper::_('searchtools.sort', 'COM_INSTALLER_HEADING_NAME', 'name', $listDirn, $listOrder); ?>
+								</th>
+								<th scope="col" style="width:10%" class="d-none d-md-table-cell">
+									<?php echo HTMLHelper::_('searchtools.sort', 'COM_INSTALLER_HEADING_LOCATION', 'client_translated', $listDirn, $listOrder); ?>
+								</th>
+								<th scope="col" style="width:10%" class="d-none d-md-table-cell">
+									<?php echo HTMLHelper::_('searchtools.sort', 'COM_INSTALLER_HEADING_TYPE', 'type_translated', $listDirn, $listOrder); ?>
+								</th>
+								<th scope="col" style="width:10%" class="d-none d-md-table-cell">
+									<?php echo HTMLHelper::_('searchtools.sort', 'COM_INSTALLER_HEADING_FOLDER', 'folder_translated', $listDirn, $listOrder); ?>
+								</th>
+								<th scope="col" style="width:5%" class="d-none d-md-table-cell">
+									<?php echo HTMLHelper::_('searchtools.sort', 'JGRID_HEADING_ID', 'update_site_id', $listDirn, $listOrder); ?>
+								</th>
+								<th scope="col" style="width:5%" class="d-none d-md-table-cell">
+									<?php echo Text::_('COM_INSTALLER_HEADING_DOWNLOADKEY'); ?>
+								</th>
+							</tr>
+						</thead>
+						<tfoot>
+							<tr>
+								<td colspan="8">
+									<?php echo $this->pagination->getListFooter(); ?>
+								</td>
+							</tr>
+						</tfoot>
+						<tbody>
+						<?php foreach ($this->items as $i => $item) : ?>
+							<tr>
+								<td class="text-center">
+									<?php if ($item->extra_query['valid']) : ?>
+										<?php echo HTMLHelper::_('grid.id', $i, $item->update_site_id); ?>
+									<?php endif; ?>
+								</td>
+								<th scope="row">
+									<?php if ($item->extra_query['valid']) : ?>
+										<a href="<?php echo Route::_('index.php?option=com_installer&task=downloadkey.edit&update_site_id=' . $item->update_site_id); ?>">
+											<?php echo $item->update_site_name; ?>
+										</a>
+									<?php else: ?>
+										<?php echo $item->update_site_name; ?>
+									<?php endif; ?>
+									<br>
+									<span class="small break-word">
+										<a href="<?php echo $item->location; ?>" target="_blank" rel="noopener noreferrer"><?php echo $this->escape($item->location); ?></a>
+									</span>
+								</th>
+								<td class="hidden-sm-down text-center">
+									<span class="bold hasTooltip" title="<?php echo HTMLHelper::_('tooltipText', $item->name, $item->description, 0); ?>">
+										<?php echo $item->name; ?>
+									</span>
+								</td>
+								<td class="hidden-sm-down text-center">
```

Comment:
Should be left aligned like the header row
```suggestion
								<td class="hidden-sm-down">
```